environment/src/main/java/com/sequenceiq/environment/parameters/dao/domain/AzureParameters.java

```diff
@@ -36,6 +36,9 @@
     @SecretValue
     private Secret encryptionKeyUrl = Secret.EMPTY;
```

Comment:
Huh, I just noticed the column name is `encryption_keyUrl`. It probably should be called `encryption_key_url` to be consistent with the rest of this class. The only issue is backward compatibility; has this column addition been already released?

To answer my question: The column was added in #10177 (CB  2.40), and the `implements AccountIdAwareResource` part was fixed in #10308 (CB  2.41). Looks like there is no way to remove / rename that column anymore. 😞 

CC @doktoric , @biharitomi, @topolyai5, @bergerdenes

Reply:
Maybe to drop column and add updated one?

Reply:
Dropping / renaming columns in general is not allowed for backwards compatibility. That is what the  schema-compatibility-test step is checking for. This case, however, is very special as the new column added earlier has never been actually in use so far. I let CB folks decide on its fate.

Reply:
@juhi-09 create a new column with name encryption_key_url and then in the next CB release you can drop the current column.

Reply:
Adding a new column `encryption_key_url` as part of this PR. Will drop `encryption_keyUrl` in future release